### DISCUSSION TITLE
Failed to install agent on Windows server 2016

### DISCUSSION THREAD

**Discussion by hanpeh2u** [immense/Remotely#680] [Q&A]
Due to unknown we failed to install agent on Windows server 2016. From the windows event we found an error below. Any idea what is missing or need to be update? 

Application: Remotely_Installer.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.IO.FileNotFoundException
   at Remotely.Agent.Installer.Win.ViewModels.MainWindowViewModel.ExtractEmbeddedServerData()
   at Remotely.Agent.Installer.Win.ViewModels.MainWindowViewModel..ctor()

**Comment by bitbound** [comment]
The next part of that stack trace should show an inner exception of type `XamlParseException` from `System.Windows.Markup.WpfXamlLoader.Load`.

This means the .NET Framework version is outdated.  You need to install the latest from https://dotnet.microsoft.com/en-us/download/dotnet-framework/net48.

**Comment by hanpeh2u** [comment]
Thank you very much!
